content/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax.md

```diff
@@ -348,6 +348,13 @@ Footnotes are not supported in wikis.
 
 **Note**: The Markdown syntax used during the beta testing period is now deprecated and will be removed. You can use the syntax as described in this section.
 
+The deprecated syntax that was previously used is as follow:
+
+```markdown
+> **Note**
+> This is a note
+```
+
 {% endnote %}
 
 Alerts are an extension of the blockquote syntax that you can use to emphasize critical information. On {% data variables.product.product_name %}, they are displayed with distinctive colors and icons to indicate the importance of the content.
```

Comment:
```suggestion
Alerts are an extension of the blockquote syntax that you can use to emphasize critical information. On {% data variables.product.product_name %}, they are displayed with distinctive colors and icons to indicate the importance of the content. Alert syntax is supported in:
- Discussions
- Gists
- Issues
- Markdown files
- Pull requests
- Releases

{% note %}

**Note:** When you edit an existing Markdown file to include an alert, the preview may display the raw alert syntax, but the file will display the alert correctly once you commit your changes.

{% endnote %}
```

I've made a couple small changes here:
- Since the information on support for alert syntax is important for all readers of this section, we should move it to the top of the section for visibility. I've also turned it into a list for scannability.
- Including the info about Markdown file previews near the list of places that support alert syntax makes a lot of sense! I've moved it up to keep the two together, as well as reformatted it into a [note](https://github.com/github/docs-internal/blob/main/contributing/content-style-guide.md#formatting-callouts) for extra visibility (the files on the Docs site are rendered differently than those on GitHub.com, so we use a different note syntax 🙂).